.storybook/components/LoremIpsum.js

```diff
@@ -1,30 +1,22 @@
-A card with a progress bar.
+import React from 'react';
 
-```jsx
-import ProgressBar from '../ProgressBar';
-
-<ProgressCard title="Upload progress" progress={<ProgressBar percent={66} />} />;
-```
-
-With children and a stepped progress bar.
-
-```jsx
-import Text from '../Text';
-import ProgressBar from '../ProgressBar';
-import SteppedProgressBar, { Step } from '../SteppedProgressBar';
+export default function LoremIpsum({ short, medium }) {
+  if (short) {
+    return <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>;
+  }
 
-<ProgressCard
-  title="Upload progress"
-  progress={
-    <SteppedProgressBar>
-      <Step complete />
-      <Step />
-      <Step />
-    </SteppedProgressBar>
+  if (medium) {
+    return (
+      <div style={{ marginTop: 8, marginBottom: 8 }}>
```

Comment:
can we not add margins to the `LoremIpsum`? I remember removing this because it was causing extra, undesired spacing in examples.

Reply:
I added first/last child logic.

Reply:
oh cool! not seeing it though?

Reply:
This is looking at the outdated code.

Reply:
you are right. i'm used to outdated comments getting minimized 🤣. i see the wrapper now!